src/controllers/expenses.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+
+const {
+  getExpenseById,
+  updateExpense,
+  createExpense,
+  getExpenseByTime,
+  getExpenseByUser,
+  getExpensesByCat,
+} = require('../services/expenses');
+
+const { getById } = require('../services/users');
+
+function getAllExpenses(expenses, users) {
+  function getExpenses(req, res) {
+    const {
+      userId,
+      category,
+      from,
+      to,
+    } = req.query;
+
+    const id = +userId;
+
+    if (typeof id !== 'number') {
+      res.sendStatus(400);
+
+      return;
+    }
```

Comment:
```suggestion
    const id = +userId;

    if (typeof id !== 'number') {
      res.sendStatus(400);

      return;
    }
```
в тебе id завжди буде number через 23 рядок